hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/WritableUpgradeStore.java

```diff
@@ -70,7 +72,7 @@ public void resetFileContents() {
 
     @Nullable
     public File removeIf(@NonNull Predicate<File> predicate) {
```

Comment:
Just a question, because I don't recall.  Why did we implement `WritableQueueState` for this class?

Reply:
I agree, this seems weird. `WritableQueueState` is on the same level as `WritableSingletonState` and `WritableKVState`. A store should be on a higher level, closer to the business logic.

Reply:
I removed it we had it before when we tried to implement singleton and to use queue as only state for upgrade file